routers/web/healthcheck/check.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package healthcheck
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/modules/cache"
+	"code.gitea.io/gitea/modules/json"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
+)
+
+type Status string
```

Comment:
Sorry, I think lint should have some problem. All public variables and functions should have comments.

Reply:
yes, I think so.

but  now I realized that I should not expose those types . so, the comments can just be avoid. -_- 

pushed, PTAL